### DISCUSSION TITLE
【UI5-Icon】ui5 icons display a bit different in horizon theme and Fiori theme

### DISCUSSION THREAD

**Discussion by GuoweiShan** [UI5/webcomponents#8212] [Q&A]
I meet an issue that, in our IRM system, the same icon name display different in two theme. For example, icon "addresses" display like this in horizon theme
<img width="40" alt="Screenshot 2024-01-31 at 16 02 25" src="https://github.com/SAP/ui5-webcomponents-react/assets/147794594/a5f1501b-6e4d-4b71-b78f-61d0a03fd701">
but it display like this in Fiori theme, 
<img width="28" alt="Screenshot 2024-01-31 at 16 03 57" src="https://github.com/SAP/ui5-webcomponents-react/assets/147794594/f4eca28b-e08e-4eeb-9ff4-a31d2842a4dd">
Is that UI5 use different versions of icon library in these two themes. Is there any solution or way to configure and custimize that use the same verison of icon libraries in these two themes

**Comment by Lukas742** [comment]
Hi @GuoweiShan 

I'll transfer this question to our UI5 Web Components Colleagues as icons are implemented by them.

**Comment by pskelin** [comment]
This is by design, here is a quote from the design guidelines:
> SAP icons have been redesigned for the Horizon visual themes with a fresh, friendly, yet bolder style that is consistent in terms of size, stroke, and balance.
https://experience.sap.com/fiori-design-web/iconography-horizon/

Here is the Quarz theme design guideline:
https://experience.sap.com/fiori-design-web/icons/

The UI5 webcomponents, as well as openui5/sapui5 automatically switch the icons according to the current theme.

If you want to force the usage of a single icon set, the README of the icons packages has more information:
`<ui5-icon name="SAP-icon-v5/accept"></ui5-icon>`
https://github.com/SAP/ui5-webcomponents/tree/main/packages/icons#collections-sap-icon-v4-and-sap-icon-v5

**Reply by GuoweiShan** [reply]
OK, thanks for your quick reply. I will have a try. Thanks a lot~
